modin/engines/ray/pandas_on_ray/frame/axis_partition.py

```diff
@@ -74,17 +74,16 @@ def deploy_axis_func(
             A list of ``pandas.DataFrame``-s.
         """
         lengths = kwargs.get("_lengths", None)
-        return deploy_ray_func._remote(
-            args=(
-                PandasFrameAxisPartition.deploy_axis_func,
-                axis,
-                func,
-                num_splits,
-                kwargs,
-                maintain_partitioning,
-            )
-            + tuple(partitions),
-            num_returns=num_splits * 4 if lengths is None else len(lengths) * 4,
+        return deploy_ray_func.options(
+            num_returns=num_splits * 4 if lengths is None else len(lengths) * 4
```

Comment:
```suggestion
            num_returns=(num_splits if lengths is None else len(lengths)) * 4
```
I wonder if this is more readable...

Reply:
@vnlitvinov , since this is not seemed to relate to the issue I would prefer to change it as part of #2919 , for example. If you don't have any other comments, feel free to merge this PR.

Reply:
absolutely possible, that's why I stamped my approval :)